### PR TITLE
Trim char for Acquisition referer

### PR DESCRIPTION
**Pull request by ydruffin-cb**
closes https://github.com/Stand-With-Crypto/swc-web/issues/676
fixes https://stand-with-crypto.sentry.io/issues/5067506303/?referrer=github_integration

**What changed? Why?**
Trim the acquisition referer when greater than 191 characters. 

**Notes to reviewers**

**How has it been tested?**

- [x] Locally
- [x] Vercel Preview Branch
- [x] Unit test

**Change management**
type=routine 
risk=low 
impact=sev5

### CODE REVIEW

File: src/utils/server/serverLocalUser.ts [hunk 1]
```diff
@@ -57,7 +57,7 @@ export function mapLocalUserToUserDatabaseFields(
   'acquisitionReferer' | 'acquisitionSource' | 'acquisitionMedium' | 'acquisitionCampaign'
 > {
   return {
-    acquisitionReferer: localUser?.persisted?.initialReferer || '',
+    acquisitionReferer: localUser?.persisted?.initialReferer?.slice(0, 191) || '',
     acquisitionSource:
```
Comment (travisbloom-cb) [inline src/utils/server/serverLocalUser.ts:61]: we prob want to trim these as well

File: src/bin/smokeTests/onNewLogin/utils.ts [hunk 2]
```diff
@@ -25,6 +26,30 @@ export function getDefaultParameters(): Params {
     injectedFetchEmbeddedWalletMetadataFromThirdweb: () => Promise.resolve(null),
   }
 }
+
+export function getParametersWithLocalUser(option?: {
```
Comment (travisbloom-cb) [inline src/bin/smokeTests/onNewLogin/utils.ts:30]: Not really sure if its work having this as an abstracted util fn if its logic for a specific test case

**Review by travisbloom-cb — CHANGES_REQUESTED** [review]

File: src/utils/server/serverLocalUser.ts (hunk 1 above)
Comment (ydruffin-cb) [reply src/utils/server/serverLocalUser.ts:61]: ✔️

File: src/bin/smokeTests/onNewLogin/utils.ts (hunk 2 above)
Comment (ydruffin-cb) [reply src/bin/smokeTests/onNewLogin/utils.ts]: ✔️

File: src/utils/server/serverLocalUser.ts [hunk 3]
```diff
@@ -57,10 +57,10 @@ export function mapLocalUserToUserDatabaseFields(
...
-      localUser?.persisted?.initialSearchParams.utm_source ||
-      localUser?.currentSession.searchParamsOnLoad.utm_source ||
+      localUser?.persisted?.initialSearchParams.utm_source?.slice(0, 191) ||
+      localUser?.currentSession.searchParamsOnLoad.utm_source?.slice(0, 191) ||
       '',
     acquisitionMedium:
```
Comment (travisbloom-cb) [inline src/utils/server/serverLocalUser.ts:65]: `acquisitionMedium` and `acquisitionCampaign`

Comment (travisbloom-cb) [reply src/utils/server/serverLocalUser.ts:65]: We prob also want a comment explaining why we're doin g this

**Review by travisbloom-cb — CHANGES_REQUESTED** [review]

**Review by travisbloom — APPROVED** [review]

**Review by travisbloom-cb — APPROVED** [review]

**Pull request merged**